burn-tensor/src/tensor/ops/tensor.rs

```diff
@@ -858,6 +858,17 @@ pub trait TensorOps<B: Backend> {
     /// A tensor with the same shape as `tensor` with square root values.
     fn sqrt<const D: usize>(tensor: B::TensorPrimitive<D>) -> B::TensorPrimitive<D>;
 
+    /// Returns a new tensor with absolute values.
+    ///
+    /// # Arguments
+    ///
+    /// * `tensor` - The tensor to exponentiate.
```

Comment:
Probably you meant to replace the word "exponentiate"

Reply:
Yes, sorry, I used different methods as templates and I forgot to change that

Reply:
Fixed